### PR TITLE
feat: About Us

### PR DESCRIPTION
**Pull request by v-almonacid**
image

### CODE REVIEW

File: packages/frontend/src/containers/header/profile-dropdown/ProfileDropdownWrapper.tsx [hunk 1]
```diff
@@ -36,14 +37,15 @@ const ProfileDropdownWrapper: React.FC<IProps> = ({
     onSignUpSignIn,
     onShowTutorial,
     showTutorial,
+    onShowAboutUsModal,
     isAuthenticated,
     setTutFocusElemRef,
     profile,
 }) => {
-    const [toggleState, setToggleState] = useState(false);
+    const [toggleTutorialState, setToggleTutorialState] = useState(false);
```
Comment (v-almonacid) [inline packages/frontend/src/containers/header/profile-dropdown/ProfileDropdownWrapper.tsx:45]: just some unrelated renaming

File: packages/ui-kit/index.ts [hunk 2]
```diff
@@ -143,6 +149,10 @@ export {
     Dialog,
     TextOverlay,
     Modal,
+    ModalHeader,
+    ModalBody,
+    ModalTitle,
+    ModalFooter,
```
Comment (v-almonacid) [inline packages/ui-kit/index.ts:155]: looks like we forgot to export these

File: packages/frontend/src/containers/AboutUsModalContainer.tsx [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+import { AboutUsModal } from "src/components/AboutUsModal";
+
+export const AboutUsModalContainer = () => {
+    const dispatch = useAppDispatch();
+    const showModal = useAppSelector((state) => state.ui.showAboutModal);
+    const toggleModal = () => dispatch(toggleAboutModal());
+
+    const onClose = () => {
+        if (showModal) {
+            toggleModal();
```
Comment (elcharitas) [inline packages/frontend/src/containers/AboutUsModalContainer.tsx:12]: Any reason why we're persisting this to the state and not using something more transient?

Comment (v-almonacid) [reply packages/frontend/src/containers/AboutUsModalContainer.tsx:12]: This doesn't need persistence, but to keep the code well organised it does make sense to keep the state in redux. It just happens that the whole redux store is persisted. 
Something similar applies to UI state variables like `showWidgetLib`

File: packages/frontend/src/components/AboutUsModal.tsx [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+                        rel="noreferrer"
+                    >
+                        alphaday.com
+                    </a>
+                    <br />
+                    API:{" "}
+                    <a
+                        href="http://api.alphaday.com/"
+                        target="_blank"
+                        rel="noreferrer"
```
Comment (elcharitas) [inline packages/frontend/src/components/AboutUsModal.tsx:83]: Please add a `"nofollow"` as well

File: packages/frontend/src/components/AboutUsModal.tsx [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+} from "@alphaday/ui-kit";
+
+interface IProps {
+    showModal: boolean;
+    onClose?: () => void;
+}
+export const AboutUsModal: React.FC<IProps> = ({ showModal, onClose }) => {
+    return (
+        <Modal showModal={showModal} onClose={onClose}>
+            <ModalHeader className="">
```
Comment (elcharitas) [inline packages/frontend/src/components/AboutUsModal.tsx:17]: Any reason why we have an empty className prop here as well?

**Review by elcharitas — APPROVED** [review]
Code looks good, lgtm

File: packages/frontend/src/components/AboutUsModal.tsx (hunk 4 above)
Comment (v-almonacid) [reply packages/frontend/src/components/AboutUsModal.tsx:83]: not sure if I get why (?)

Comment (v-almonacid) [reply packages/frontend/src/components/AboutUsModal.tsx:83]: I see what you mean, it makes sense, but this html shouldn't be rendered to crawlers. Also, I'm waiting for Deniz's response, because I'd prefer to remove it.

File: packages/frontend/src/components/AboutUsModal.tsx [hunk 6]
```diff
@@ -0,0 +1,111 @@
...
+    ModalTitle,
+} from "@alphaday/ui-kit";
+
+interface IProps {
+    showModal: boolean;
+    onClose?: () => void;
+}
+export const AboutUsModal: React.FC<IProps> = ({ showModal, onClose }) => {
+    return (
+        <Modal showModal={showModal} onClose={onClose}>
```
Comment (Xavier-Charles) [inline packages/frontend/src/components/AboutUsModal.tsx:16]: Please set modal size to "md" so that it doesn't look stretched.
Screenshot 2024-08-08 at 21 29 53

File: packages/ui-kit/index.ts (hunk 2 above)
Comment (Xavier-Charles) [reply packages/ui-kit/index.ts:155]: That's because Modals were placed here  `packages/ui-kit/src/components/modal`
Screenshot 2024-08-08 at 21 33 46

**Review by Xavier-Charles — APPROVED** [review]
Looks good added a few comments.
